src/checks/unordered_key.rs

```diff
@@ -369,4 +374,48 @@ mod tests {
 
         run_unordered_tests(asserts);
     }
+
+    #[test]
+    fn one_unordered_key_with_blank_line_test() {
+        let asserts = vec![
+            (
+                LineEntry {
+                    number: 1,
+                    file: FileEntry {
+                        path: PathBuf::from(".env"),
+                        file_name: ".env".to_string(),
+                        total_lines: 2,
+                    },
+                    raw_string: String::from("FOO=BAR"),
+                },
+                None,
+            ),
+            (
+                LineEntry {
+                    number: 1,
+                    file: FileEntry {
+                        path: PathBuf::from(".env"),
+                        file_name: ".env".to_string(),
+                        total_lines: 2,
+                    },
+                    raw_string: String::from(""),
+                },
+                None,
+            ),
+            (
+                LineEntry {
+                    number: 2,
```

Comment:
Shouldn't that be three?

```suggestion
                    number: 3,
```